### ISSUE TITLE
Provide source maps support

### ISSUE THREAD

**Issue by sentika** [peerigon/extract-loader#1]
There is no source map after using extract. For example in this chain:
style/url!file!extract!css

**Comment by jhnns** [comment]
That's true. Source maps are currently not supported. I'm working on that.

**Comment by RobertoUa** [comment]
How long do you think it's gonna take?

**Comment by jhnns** [comment]
I'm sorry, too much work currently. If anyone is interested, please go ahead :). The code is fairly simple and I can give you support.

**Comment by ukch** [comment]
@jhnns I can work on it if you show me where to start. (I'm pretty new to Webpack development though)

**Comment by wujekbogdan** [comment]
@sentika

This is my scss compilation/extraction section of my webpack config file:

``` javascript
{
    test: /\.scss$/,
    use: [
        {
            loader: 'file-loader',
            options: {
                name: '[name].css',
                context: './src/css/',
                outputPath: 'css/',
                publicPath: '../'
            }
        },
        {
            loader: 'extract-loader'
        },
        {
            loader: 'css-loader',
            options: {
                sourceMap: true
            }
        },
        {
            loader: 'resolve-url-loader'
        },
        {
            loader: 'postcss-loader'
        },
        {
            loader: 'sass-loader',
            options: {
                sourceMap: true
            }
        }
    ]
}
```

And I can see sourcemaps, but instead of separate `.map` files (as they should be because of `devtool: '#source-map'` in the configuration file) they are merged into `*.css` files at the end of `*.css` files. There's an another side-effect: paths in sourcemaps are absolute (`c:/your/project/directory/file-name-css`)

It's a workaround, not a solution, but at least works.

**Comment by wearemetabolism** [comment]
Is there any ETA ?

**Comment by jhnns** [comment]
@jstrimpel I tried to explain it to you how to fix it. On the way explaining it, I found the solution for the problem 😁 

Shipped with `v3.0.0` 🚀

**Comment by jstrimpel** [comment]
@jhnns Thank you so much!
